src/core/data/ions/particle_initializers/particle_initializer_factory.hpp

```diff
@@ -47,13 +47,9 @@ namespace core
 
                 auto charge = dict["charge"].template to<double>();
 
-                auto nbrPartPerCell = cppdict::get_value(dict, "nbr_part_per_cell", int{0});
-                FunctionType nbrPartPerCellFn
-                    = cppdict::get_value(dict, "nbr_part_per_cell_fn", FunctionType{nullptr});
-                if (not nbrPartPerCellFn and nbrPartPerCell == 0)
-                {
-                    throw std::runtime_error("PPC cannot be 0");
-                }
+                auto densityCutOff = cppdict::get_value(dict, "density_cut_off", double{1e-5});
```

Comment:
of it's in the dict already, I got confused why there was another 1e-5 earlier then

Reply:
it's not in the dict, still todo from python